contracts/MarginBase.sol

```diff
@@ -177,36 +179,35 @@ contract MarginBase is MinimalProxyable, IMarginBase, OpsReady {
                                 Views
     ///////////////////////////////////////////////////////////////*/
 
-    /// @notice get number of active market positions account has
-    /// @return number of positions which are currently active for account
-    function getNumberOfActivePositions() external view returns (uint256) {
+    /// @notice get number of internal market positions account has
+    /// @return number of positions which are internally accounted for
+    function getNumberOfInternalPositions() external view returns (uint256) {
         return activeMarketKeys.length;
     }
 
-    /// @notice get all active market positions
-    /// @return positions which are currently active for account (ActiveMarketPosition structs)
-    function getAllActiveMarketPositions()
```

Comment:
Maybe not *as* useful. 

You can see how many active market positions there are via `getNumberOfInternalPositions()` and FE can iterate over those indices (ex: `activeMarketKeys[0], activeMarketKeys[1], ...) calling `getPosition()`

It may not be a bad thing to add back if you think it will be beneficial for the FE. I was just trying to reduce code that wasn't 100% necessary